example/migrations/0001_initial.py

```diff
@@ -1,37 +1,176 @@
-# Generated by Django 3.2.8 on 2021-10-18 10:59
```

Comment:
**Question**: Have you squashed all the previous migrations into one? If so, good shout.

Reply:
yes I did 😄